File: Estrutura_Condicional/C++/Ex1038/main.cpp
#include <iostream>
#include <iomanip>
#include <string>

int main() {
    int cod, qnt;
    std::cin >> cod >> qnt;

    if (cod == 1) {
        std::cout << std::fixed << std::setprecision(2) << "Total: R$ " << qnt * 4.00 << std::endl;
    } else if (cod == 2) {
        std::cout << std::fixed << std::setprecision(2) << "Total: R$ " << qnt * 4.50 << std::endl;
    } else if (cod == 3) {
        std::cout << std::fixed << std::setprecision(2) << "Total: R$ " << qnt * 5.00 << std::endl;
    } else if (cod == 4) {
        std::cout << std::fixed << std::setprecision(2) << "Total: R$ " << qnt * 2.00 << std::endl;
    } else if (cod == 5) {
        std::cout << std::fixed << std::setprecision(2) << "Total: R$ " << qnt * 1.50 << std::endl;
    }

    return 0;
}

File: Estrutura_Condicional/Java/Ex1038/Main.java
package Ex1038;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cod, qnt;
        cod = sc.nextInt();
        qnt = sc.nextInt();
        sc.close();

        if (cod == 1){
            System.out.printf("Total: R$ %.2f\n", qnt * 4.00);
        } else if (cod == 2){
            System.out.printf("Total: R$ %.2f\n", qnt * 4.50);
        } else if (cod == 3){
            System.out.printf("Total: R$ %.2f\n", qnt * 5.00);
        } else if (cod == 4){
            System.out.printf("Total: R$ %.2f\n", qnt * 2.00);
        } else if (cod == 5){
            System.out.printf("Total: R$ %.2f\n", qnt * 1.50);
        }
    }
}